serpent-moltres/fuel-column-sensitivity/extract-convert.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python3
+# This file is based in $MOLTRES/python/extractSerpent2GCs.py
```

Comment:
Please improve that file rather than duplicating it here. This is like a mini hostile fork of the file in Moltres. I will not merge it.